src/test/java/org/bigml/binding/CommonStepdefs.java

```diff
@@ -265,7 +265,7 @@ public void the_resource_has_been_updated_with_status(int status) {
     @Then("^delete all test data$")
     public void delete_all_test_data() throws Exception {
 
-        BigMLClient.getInstance().getCacheManager().cleanCache();
+        //BigMLClient.getInstance().getCacheManager().cleanCache();
```

Comment:
is this a left-off or what is the reason for it?

Reply:
Same reason than other comment